docs/light/api.md

```diff
@@ -565,11 +564,65 @@ Returns on failure:
 }
 ```
 
-### /stake/delegators/{delegatorAddr}/delegations - POST
+### POST /stake/delegators/{delegatorAddr}/delegations
+
+url: `/stake/delegators/{delegatorAddr}/delegations`
+
+Functionality: Submit or edit a delegation.
+
+> NOTE: Should this be a PUT instead of a POST? the code indicates that this is an edit operation
```

Comment:
@jackzampolin I think the idea is to split this function into several ones, but I agree